### PR TITLE
Update telemed vids

### PR DESCRIPTION
**Pull request by harimohanraj89**
**Story card:** [ch2585](https://app.clubhouse.io/simpledotorg/story/2585/update-teleconsultation-help-video)

## Because

The telemed training videos in the app and dashboard were outdated

## This addresses

Updates the videos to the new versions. Touchpoints:
* Help section in the App
* Resources section in the Dashboard

### CODE REVIEW

File: config/locales/api/help/en.html [hunk 1]
```diff
@@ -114,7 +114,7 @@ <h3>Class chapters</h3>
             <h3>Tele-consultation training</h3>
             <p>If your hospital is using Simple for tele-consultation, this training is for you.</p>
             <div class="video-small">
-                <a href="https://youtu.be/LpLeFFKhXd4">Tele-consultation with Simple<span>English</span></a>
+                <a href="https://youtu.be/aI1bGpS08oY">Tele-consultation with Simple<span>English</span></a>
             </div>
             <div class="video-small">
                 <a href="https://youtu.be/lMS1GiqbKps">सिम्पल द्वारा टेली-मेडिसिन<span>Hindi</span></a>
```
Comment (prabhanshuguptagit) [inline config/locales/api/help/en.html:120]: Did this get missed 👾 ?

File: config/locales/api/help/en.html [hunk 2]
```diff
@@ -114,7 +114,7 @@ <h3>Class chapters</h3>
             <h3>Tele-consultation training</h3>
             <p>If your hospital is using Simple for tele-consultation, this training is for you.</p>
             <div class="video-small">
-                <a href="https://youtu.be/LpLeFFKhXd4">Tele-consultation with Simple<span>English</span></a>
+                <a href="https://youtu.be/aI1bGpS08oY">Tele-consultation with Simple<span>English</span></a>
```
Comment (prabhanshuguptagit) [inline config/locales/api/help/en.html:117]: Should this be `ExternalLinksHelper::TELEMEDICINE_TRAINING_ENGLISH_YOUTUBE_LINK`?

File: config/locales/api/help/en.html (hunk 1 above)
Comment (harimohanraj89) [reply config/locales/api/help/en.html]: Indeed.

**Comment by harimohanraj89** [comment]
@prabhanshuguptagit Ready for 👁️ 👃 👁️

**Review by prabhanshuguptagit — APPROVED** [review]
`rspec` not found, weird spec failure.

**Comment by harimohanraj89** [comment]
Ugh I do not know why that's happening. This'll take a bit of time to diagnose/rectify.

**Comment by harimohanraj89** [comment]
Looks like this branch is just borked on Semaphore. I confirmed that the tests pass in a fresh branch: https://simple.semaphoreci.com/workflows/2eebadf6-0023-4f20-8e2f-7af076101bcc?pipeline_id=448c6655-9063-43f3-9d13-f593b140d0fd

**Comment by harimohanraj89** [comment]
Closed in favor of #2101

**Pull request closed without merging**